docs/howto/openstack/nova/new-vgpu-server.md

```diff
@@ -0,0 +1,170 @@
+# Creating new servers with virtual GPU support
+
+In {{brand}}, you can instantiate virtual machines --- or simply
+_servers_ --- with virtual GPU (vGPU) support. For that, you use the
+`openstack` CLI utility.
+
+## Prerequisites
+
+Although not strictly necessary, it is certainly helpful to have some
+familiarity with [creating servers in {{brand}}](new-server.md),
+especially with the OpenStack CLI.
+
+## Instantiating a vGPU server
+
+Before you begin, please make sure to source [your `openstack` RC file
+first](../../getting-started/enable-openstack-cli.md).
+
+As we discussed in the [guide on creating servers](new-server.md), an
```

Comment:
This howto guide is not a part of a sequential series, so we cannot assume as certainty that the reader has read the other guide. Thus, this feels more natural to me:
```suggestion
As we discuss in the [guide on creating servers](new-server.md), an
```

Reply:
It is more natural (and correct), yes.